docs/sidebars.js

```diff
@@ -14,57 +14,63 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
     main: [
-        'home',
+        {
+            "type": 'category',
+            label: 'Get Started',
+            collapsed: true,
+            items: [
+                {type: 'autogenerated', dirName: 'get-started'}
+            ]
+        },
         {
             "type": "link",
             "label": "Kurtosis for Web3",
             "href": "https://web3.kurtosis.com",
         },
-        'quickstart',
-        'code-examples',
         {
             type: 'category',
-            label: 'Guides',
```

Comment:
Would prefer the `Guides` section be near the top of the sidebar, proposing that it be right below the `Quickstart`

Reply:
No issue with that will change